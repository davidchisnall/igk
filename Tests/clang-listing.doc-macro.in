\docfile{includes/test.cc}

\macrodoc{MACRO}

\macrodoc{MACRO_ARG}

