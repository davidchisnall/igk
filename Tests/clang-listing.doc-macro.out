

\clang-doc[code-declaration-entity=MACRO,code-declaration-kind=macro]{\code[code-kind=listing]{\code-run[token-kind=MacroName]{MACRO}}\p{ This is a macro}}

\clang-doc[code-declaration-entity=MACRO_ARG,code-declaration-kind=macro]{\code[code-kind=listing]{\code-run[token-kind=MacroName]{MACRO_ARG}\code-run[token-kind=Punctuation]{(}\code-run[token-kind=Identifier]{x}\code-run[token-kind=Punctuation]{,}\code-run[token-kind=Identifier]{y}\code-run[token-kind=Punctuation]{,}\code-run[token-kind=Identifier]{z}\code-run[token-kind=Punctuation]{)}}\p{This is a macro with an argument}}

